e2e/site-navigation.test.ts

```diff
@@ -0,0 +1,357 @@
+		await expect( page.locator( '#wpadminbar' ) ).toBeVisible();
+		await expect( page.locator( '#adminmenuback' ) ).toBeVisible();
+
+		// Verify we're logged in by checking for user menu
+		const userMenu = page.locator( '#wp-admin-bar-my-account' );
+		await expect( userMenu ).toBeVisible();
+	} );
+
+	test( 'creates a post', async ( { page } ) => {
+		// Navigate to new post page
+		const newPostUrl = `${ wpAdminUrl }/post-new.php`;
+		await page.goto( getUrlWithAutoLogin( newPostUrl ) );
+
+		const editorFrame = page.frameLocator( 'iframe[name="editor-canvas"]' );
+
+		// Close welcome guide if it appears (always on main page, not in iframe)
+		await closeWelcomeGuide( page );
+
+		// Wait for title to be available in iframe
+		const titleSelector = 'h1.editor-post-title__input';
+		await editorFrame.locator( titleSelector ).waitFor( { timeout: 10_000 } );
+		await editorFrame.locator( titleSelector ).fill( 'E2E Test Post' );
+
+		// Click into the content area and type
+		await editorFrame.locator( titleSelector ).press( 'Enter' );
+		const contentBlock = editorFrame.locator( 'p[role="document"]' ).first();
+		await contentBlock.fill( 'This is a test post created by automated E2E tests.' );
+
+		// Publish the post (publish buttons are on main page)
+		const publishButton = page.locator(
+			'button.editor-post-publish-button__button.editor-post-publish-panel__toggle'
+		);
+
+		await publishButton.waitFor( { state: 'visible', timeout: 10_000 } );
+		await publishButton.click();
+
+		// Wait for and click the confirm publish button in the panel
+		const confirmPublishButton = page.locator(
+			'.editor-post-publish-panel__header-publish-button button.editor-post-publish-button__button'
+		);
+		await confirmPublishButton.waitFor( { state: 'visible', timeout: 10_000 } );
+		await confirmPublishButton.click();
+
+		// Wait for success message
+		await expect( page.locator( '.components-snackbar' ) ).toBeVisible( { timeout: 10_000 } );
+
+		// Verify post was created by visiting posts list
+		await page.goto( getUrlWithAutoLogin( `${ wpAdminUrl }/edit.php` ) );
+		await expect( page.locator( 'a.row-title:has-text("E2E Test Post")' ) ).toBeVisible();
+	} );
+
+	test( 'uploads media', async ( { page } ) => {
+		// Navigate to media library
+		const addNewMediaUrl = `${ wpAdminUrl }/media-new.php`;
+		await page.goto( getUrlWithAutoLogin( addNewMediaUrl ) );
+
+		// Create a minimal valid PNG file (1x1 red pixel)
+		const pngBuffer = Buffer.from( [
+			0x89, 0x50, 0x4e, 0x47, 0x0d, 0x0a, 0x1a, 0x0a, 0x00, 0x00, 0x00, 0x0d, 0x49, 0x48, 0x44,
+			0x52, 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x01, 0x08, 0x02, 0x00, 0x00, 0x00, 0x90,
+			0x77, 0x53, 0xde, 0x00, 0x00, 0x00, 0x0c, 0x49, 0x44, 0x41, 0x54, 0x08, 0x99, 0x63, 0xf8,
+			0xcf, 0xc0, 0x00, 0x00, 0x03, 0x01, 0x01, 0x00, 0x18, 0xdd, 0x83, 0x75, 0x00, 0x00, 0x00,
+			0x00, 0x49, 0x45, 0x4e, 0x44, 0xae, 0x42, 0x60, 0x82,
+		] );
+
+		const testImagePath = '/tmp/e2e-test-image.png';
```

Comment:
I think this is the error I'm getting on Windows, as it cannot find the `/tmp/` directory. Consider using `os.tmpdir()`

Reply:
Thanks for the great suggestion @epeicher! 💯 I applied it and it seems to work, the windows tests are now failing at a different step 🫣